### DISCUSSION TITLE
zigbee2mqtt Post Install Instructions Migrate

### DISCUSSION THREAD

**Discussion by kylezoa** [community-scripts/ProxmoxVE#407] [General]
[Zigbee2MQTT on tteck](https://tteck.github.io/Proxmox/#zigbee2mqtt-lxc) has a link to [Post Install](https://github.com/tteck/Proxmox/discussions/2917) instructions that is not on the [community-scripts](https://community-scripts.github.io/ProxmoxVE/scripts?id=zigbee2mqtt) page.

**Comment by MickLesk** [comment]
Thx! 
Added in Discussions and will be added on website asap
https://github.com/community-scripts/ProxmoxVE/discussions/410
